main.cpp

```diff
@@ -1,26 +1,39 @@
-#include "home.h"
-#include "models/mock_model.h"
-#include <QApplication>
+#include <QGuiApplication>
+#include <QQmlApplicationEngine>
+
 #include <QLocale>
+#include <QQmlContext>
 #include <QTranslator>
+#include <homecontroller.h>
 
 int main(int argc, char *argv[]) {
-  QApplication a(argc, argv);
+  QGuiApplication app(argc, argv);
 
   QTranslator translator;
```

Comment:
what's all this translator code for?

Reply:
I would add some comments to this file since setting everything up is kinda weird, but overall looks dope to me, a lot more useable than the widget stuff

Reply:
this is auto generated lmao idk